zubhub_frontend/zubhub/src/views/pageWrapperScripts.js

```diff
@@ -98,3 +98,9 @@ export const closeSearchFormOrIgnore = e => {
     handleCloseSearchForm();
   }
 };
+
+export const sliceLongName = (name) => {
+  const arr = name.split('')
+  arr.splice(11,arr.length-14,'...')
```

Comment:
Same as above :point_up_2: Maybe make it ` arr.splice(11, arr.length - 14, '...')`

Reply:
done